src/Catalyst.Core.Modules.Dfs.Tests/IntegrationTests/CoreApi/SwarmApiTest.cs

```diff
@@ -0,0 +1,200 @@
+//             Assert.True(addrs.Any(a => a == somewhere));
+//
+//             addr = await ipfs.Swarm.RemoveAddressFilterAsync(somewhere);
+//             Assert.NotNull(addr);
+//             Assert.Equal(somewhere, addr);
+//             addrs = await ipfs.Swarm.ListAddressFiltersAsync();
+//             Assert.False(addrs.Any(a => a == somewhere));
+//         }
+//
+//         // [Fact]
+//         // [Ignore("https://github.com/richardschneider/net-ipfs-engine/issues/74")]
+//         // public async Task Connect_Disconnect_Mars()
+//         // {
+//         //     var mars = "/dns/mars.i.ipfs.io/tcp/4001/ipfs/QmaCpDMGvV2BGHeYERUEnRQAwe3N8SzbUtfsmvsqQLuvuJ";
+//         //     await ipfs.StartAsync();
+//         //     try
+//         //     {
+//         //         await ipfs.Swarm.ConnectAsync(mars);
+//         //         var peers = await ipfs.Swarm.PeersAsync();
+//         //         Assert.True(peers.Any(p => p.Id == "QmaCpDMGvV2BGHeYERUEnRQAwe3N8SzbUtfsmvsqQLuvuJ"));
+//         //         await ipfs.Swarm.DisconnectAsync(mars);
+//         //     }
+//         //     finally
+//         //     {
+//         //         await ipfs.StopAsync();
+//         //     }
+//         // }
+//
+//         // [Fact]
+//         // [Ignore("TODO: Move to interop tests")]
+//         // public async Task JsIPFS_Connect()
+//         // {
+//         //     var cts = new CancellationTokenSource(TimeSpan.FromSeconds(10));
+//         //     var remoteId = "QmXFX2P5ammdmXQgfqGkfswtEVFsZUJ5KeHRXQYCTdiTAb";
+//         //     var remoteAddress = $"/ip4/127.0.0.1/tcp/4002/ipfs/{remoteId}";
+//         //
+//         //     Assert.Equal(0, (await ipfs.Swarm.PeersAsync()).Count());
+//         //     await ipfs.Swarm.ConnectAsync(remoteAddress, cts.Token);
+//         //     Assert.Equal(1, (await ipfs.Swarm.PeersAsync()).Count());
+//         //
+//         //     await ipfs.Swarm.DisconnectAsync(remoteAddress);
+//         //     Assert.Equal(0, (await ipfs.Swarm.PeersAsync()).Count());
+//         // }
+//         //
+//         // [Fact]
+//         // [Ignore("TODO: Move to interop tests")]
+//         // public async Task GoIPFS_Connect()
+//         // {
+//         //     var cts = new CancellationTokenSource(TimeSpan.FromSeconds(10));
+//         //     var remoteId = "QmdoxrwszT6b9srLXHYBPFVRXmZSFAosWLXoQS9TEEAaix";
+//         //     var remoteAddress = $"/ip4/127.0.0.1/tcp/4001/ipfs/{remoteId}";
+//         //
+//         //     Assert.Equal(0, (await ipfs.Swarm.PeersAsync()).Count());
+//         //     await ipfs.Swarm.ConnectAsync(remoteAddress, cts.Token);
+//         //     Assert.Equal(1, (await ipfs.Swarm.PeersAsync()).Count());
+//         //
+//         //     await ipfs.Swarm.DisconnectAsync(remoteAddress);
+//         //     Assert.Equal(0, (await ipfs.Swarm.PeersAsync()).Count());
+//         // }
+//         //
+//         // [Fact]
+//         // [Ignore("TODO: Move to interop tests")]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)